mongorestore/mongorestore_test.go

```diff
@@ -583,3 +583,91 @@ func TestFixHashedIndexes(t *testing.T) {
 		})
 	})
 }
+
+func TestRandom(t *testing.T) {
```

Comment:
This looks like this may have just been a placeholder name

Reply:
Oops, you're right!